app/Providers/DevDependenciesServiceProvider.php

```diff
@@ -16,7 +16,7 @@ public function boot(): void
             // To use "require-dev" dependencies, their providers should be registered via full namespace
             // TODO: restore when the dependency would be updated
             //App::register(\RonasIT\Support\EntityGeneratorServiceProvider::class);
-            App::register(\Barryvdh\LaravelIdeHelper\IdeHelperServiceProvider::class);
+            //App::register(\Barryvdh\LaravelIdeHelper\IdeHelperServiceProvider::class);
```

Comment:
please check that we really no need it this providers and autodiscovering works for it and remove it

Reply:
@DenTray Yes, the IDE Helper still generates files as before. Just in case, https://medium.com/@taylorotwell/package-auto-discovery-in-laravel-5-5-ea9e3ab20518